### ISSUE TITLE
Questioning the need for derive_param_lasta() from tester feedback

### ISSUE THREAD

**Issue by rossfarrugia** [pharmaverse/admiralonco#89]
Stefan: At Roche we would need to use the source_pd argument for the derivation of the LSTAC parameter. However, this parameter is not used anywhere. So I would question it. **(needs to be checked with Roche data standards to understand why we have this in our global standards)**
Furthermore, I would question if we need derive_param_lasta() at all. The Roche testers requested an update of derive_param_first_event() (https://github.com/pharmaverse/admiral/issues/1317) which covers the derive_param_lasta() functionality (except the source_pd part).

**Comment by bundfussr** [comment]
@rossfarrugia , I agree that we should not change it for this release. Thus I have removed the release label.

However, I doubt more and more that the `source_pd` approach is a good solution. So we may rediscuss the topic after the release.

**Comment by rossfarrugia** [comment]
Thanks @bundfussr - do you want to set up a call next week (after the first release) with us, @HinalPat, @amitjaingsk and @sgorm123 and we could consider alternative ideas you have.

**Comment by rossfarrugia** [comment]
Decision: once we have https://github.com/pharmaverse/admiral/issues/1317 updated, we can use derive_param_first_event() instead in vignette/template, and deprecate derive_param_lasta().

**Comment by rossfarrugia** [comment]
@sgorm123 FYI we expect `derive_param_extreme_event()` (https://github.com/pharmaverse/admiral/issues/1317) will get merged in {admiral} in the next week or so, so can you check if you'll have time to make this issues updates towards end of the month ready for our Dec 12th release? otherwise, we can re-assign. FYI @HinalPat

**Comment by bundfussr** [comment]
@sgorm123 , @rossfarrugia , @HinalPat , `derive_param_extreme_event()` (https://github.com/pharmaverse/admiral/issues/1317) is available in the devel version admiral now.

**Comment by rossfarrugia** [comment]
Great! If we don't hear back from Stephen I suggest opening up this issue to other admiralonco devs for someone to take this on now. @HinalPat can you arrange for this please?

**Comment by rossfarrugia** [comment]
@sgorm123 , @bundfussr , @HinalPat FYI that `derive_param_response()` uses `derive_param_first_event()` so will need to also be updated to use `derive_param_extreme_event()` instead as part of this update

**Comment by rossfarrugia** [comment]
@sgorm123 will you have time for this issue before the release scheduled for 12th Dec? if not let me know and i could pick this one up next week
